certmanager/util.go

```diff
@@ -18,10 +18,40 @@ import (
 	"crypto"
 	"crypto/x509"
 	"encoding/pem"
+	"net/http"
 
 	"github.com/golang/glog"
 )
 
+const (
+	// The GCE metadata service URI to get identity token.
```

Comment:
That's not entirely true, no?

To get token for SA 'foo', the URL needs to be service-accounts/foo/identity